### ISSUE TITLE
Flaky accessibility test build

### ISSUE THREAD

**Issue by jgao-czi** [chanzuckerberg/axe-storybook-testing#89]
We've been getting some flaky  accessibility test build failures. It passes when I re-run the build. Could someone pair with me on this? 

https://github.com/chanzuckerberg/edu-ps-intervention-tracking/actions/runs/8085775219/job/22094050662?pr=242

Here is the full log.
[7_🤟 Accessibility tests.txt](https://github.com/chanzuckerberg/axe-storybook-testing/files/14439088/7_.Accessibility.tests.txt)

**Comment by ahuth** [comment]
Yes, I can absolutely pair up on this. I'll reach out.
